src/views/Generator/TestOptions/LoadZones/LoadZoneRow.tsx

```diff
@@ -0,0 +1,69 @@
+import { FieldGroup, ControlledSelect } from '@/components/Form'
+import { Table } from '@/components/Table'
+import { LoadZoneData } from '@/types/testOptions'
+import {
+  FieldArrayWithId,
+  UseFieldArrayRemove,
+  useFormContext,
+} from 'react-hook-form'
+import { IconButton, TextField } from '@radix-ui/themes'
+import { TrashIcon } from '@radix-ui/react-icons'
+import { LOAD_ZONES_REGIONS_OPTIONS } from './LoadZones.utils'
+
+type LoadZoneRowProps = {
+  index: number
+  field: FieldArrayWithId<LoadZoneData, 'loadZones', 'id'>
+  remove: UseFieldArrayRemove
+}
+
+export function LoadZoneRow({ field, index, remove }: LoadZoneRowProps) {
+  const {
+    register,
+    formState: { errors },
+    control,
+    watch,
+  } = useFormContext<LoadZoneData>()
+
+  const { distribution, loadZones } = watch()
+
+  // Disable load zone options that are already in use
+  const getLoadZoneOptions = () => {
+    return LOAD_ZONES_REGIONS_OPTIONS.map((option) => ({
+      ...option,
+      disabled: loadZones.some(
+        (zone, i) => zone.loadZone === option.value && i !== index
+      ),
+    }))
+  }
```

Comment:
This ensures that options can't be selected more than once

![image](https://github.com/user-attachments/assets/69b5da3a-9bd6-40f3-aa28-bebedb26172d)